Source/Core/DolphinWX/Debugger/BreakpointWindow.cpp

```diff
@@ -57,33 +49,62 @@ class CBreakPointBar : public DolphinAuiToolBar
     AddTool(ID_ADDMC, "+MBP", m_Bitmaps[Toolbar_Add_MC]);
     Bind(wxEVT_TOOL, &CBreakPointWindow::OnAddMemoryCheck, parent, ID_ADDMC);
 
-    AddTool(ID_LOAD, _("Load"), m_Bitmaps[Toolbar_Delete]);
+    AddTool(ID_LOAD, _("Load"), m_Bitmaps[Toolbar_Load]);
     Bind(wxEVT_TOOL, &CBreakPointWindow::Event_LoadAll, parent, ID_LOAD);
 
-    AddTool(ID_SAVE, _("Save"), m_Bitmaps[Toolbar_Delete]);
+    AddTool(ID_SAVE, _("Save"), m_Bitmaps[Toolbar_Save]);
     Bind(wxEVT_TOOL, &CBreakPointWindow::Event_SaveAll, parent, ID_SAVE);
   }
 
+  void ReloadBitmaps()
+  {
+    Freeze();
+
+    InitialiseThemedBitmaps();
+    for (int i = 0; i < ID_NUM; ++i)
+      SetToolBitmap(i, m_Bitmaps[i]);
+
+    Thaw();
+  }
+
 private:
   enum
   {
-    Toolbar_Delete,
+    Toolbar_Delete = 0,
+    Toolbar_Clear,
     Toolbar_Add_BP,
     Toolbar_Add_MC,
+    Toolbar_Load,
+    Toolbar_Save,
     Num_Bitmaps
   };
 
   enum
   {
-    ID_DELETE = 2000,
+    ID_DELETE = 0,
     ID_CLEAR,
     ID_ADDBP,
     ID_ADDMC,
     ID_LOAD,
-    ID_SAVE
+    ID_SAVE,
+    ID_NUM
   };
 
+  void InitialiseThemedBitmaps()
+  {
+    wxSize bitmap_size = FromDIP(wxSize(24, 24));
+    SetToolBitmapSize(bitmap_size);
+
+    for (std::size_t i = 0; i < m_image_names.size(); ++i)
+      m_Bitmaps[i] =
```

Comment:
This should be braced

Reply:
@aldelaro5 You forgot to address this comment as well.